deps/pylib.sh

```diff
@@ -1 +1,2 @@
 pip install requests_toolbelt filemagic ftputil
+pip install git+https://github.com/metabolite-atlas/metatlas
```

Comment:
And let's check if metatlas was already installed. So instead of
```
pip install git+https://github.com/metabolite-atlas/metatlas
```
we can put this:
```
python_metatlas_installed=`pip list | grep '^metatlas (' | wc -l`
if [ "$python_metatlas_installed" == "0" ]; then
    pip install git+https://github.com/metabolite-atlas/metatlas
fi
```